broker-node/app/Clients/iota-php-port/PrepareTransfers.php

```diff
@@ -24,8 +24,15 @@ public static function prepareTransfers($seed, $transfers, $options = null, $cal
         // Also remove the checksum of the address if it's there after validating it
         foreach ($transfers as $thisTransfer) {
 
-            $thisTransfer->message = $thisTransfer->message ? $thisTransfer->message : '';
-            $thisTransfer->obsoleteTag = $thisTransfer->tag ? $thisTransfer->tag : ($thisTransfer->obsoleteTag ? $thisTransfer->obsoleteTag : '');
```

Comment:
Thanks for expanding this.
I don't mind the short if/then notation if it is just one but nested it gets hard to understand at a glance.